enhancements/baremetal/baremetal-provisioning-optional.md

```diff
@@ -60,9 +60,12 @@ entirely.
 
 ### Non-Goals
 
-We do not intend to remove the static provisioning IP's at this time,
+- We do not intend to remove the static provisioning IP's at this time,
 but instead they will need to be IP's available in the external network.
 
+- As is the case for any other provisioning network decisions, this is a
+day 1 choice that we do not intend to support changing at a later time.
```

Comment:
We know there are some requirements under discussion for this, so maybe we could phrase this as "we do not currently intend to support..." and mention that any work to enable the network as a day-2 choice would be discussed in a separate/future enhancement proposal?